scripts/js/lib/extractMarkdownImages.ts

```diff
@@ -0,0 +1,39 @@
+// This code is a Qiskit project.
+//
+// (C) Copyright IBM 2024.
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+import { unified } from "unified";
+import { Root } from "remark-mdx";
+import { visit } from "unist-util-visit";
+import remarkParse from "remark-parse";
+import remarkGfm from "remark-gfm";
+import remarkStringify from "remark-stringify";
+
+export async function findImagesWithoutAltText(
+  markdown: string,
+): Promise<Set<string>> {
+  const images = new Set<string>();
+
+  await unified()
+    .use(remarkParse)
+    .use(remarkGfm)
+    .use(() => (tree: Root) => {
+      visit(tree, "image", (node) => {
+        if (!node.alt) {
+          images.add(node.url);
+        }
+      });
+    })
+    .use(remarkStringify)
+    .process(markdown);
```

Comment:
Notice that the plugin will not extract images from HTML tags. I have searched in the repo and we don't have any instance of it. HTML tags can be used for the writers if they want to resize the images in markdown:
```html
<img src="example.jpg" alt="Example" width="200"/>
```
We could implement this by adding another visitor that uses Cheerio as follows:
```ts
visit(tree, "html", (node) => {
  const $ = load(node.value);
  const imgElement = $("img");
  if (imgElement.length) {
    images.add(imgElement.attr("src"));
 }
});
```

Do you think we should take into account this case @eric-arellano?